src/main/test/com/social/AttachmentsTest.java

```diff
@@ -1,21 +1,30 @@
-package com.salazart;
+package com.social;
+
+import static org.junit.Assert.*;
 
 import java.io.IOException;
 
-import org.codehaus.jackson.JsonGenerationException;
-import org.codehaus.jackson.map.JsonMappingException;
 import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Test;
 
 import com.social.models.Attachment;
 import com.social.models.Media;
 
 public class AttachmentsTest {
-	public static void main(String[] args) throws JsonGenerationException, JsonMappingException, IOException {
+	
+	@Test
+	public void test() {

```

Comment:
What are you trying to test here: jackson library? I think it is not your work ;)